packages/astro/src/core/build/index.ts

```diff
@@ -232,21 +250,13 @@ class AstroBuilder {
   }
 
   /** Stats */
-  private async printStats({ cwd, pageCount }: { cwd: URL; pageCount: number }) {
-    const [js, html] = await Promise.all([profileJS({ cwd, entryHTML: new URL('./index.html', cwd) }), profileHTML({ cwd })]);
-
+  private async printStats({ logging, timeStart, pageCount }: { logging: LogOptions; timeStart: number; pageCount: number }) {
     /* eslint-disable no-console */
-    console.log(`${bold(cyan('Done'))}
-Pages (${pageCount} total)
-  ${green(`✔ All pages under ${kb(html.maxSize)}`)}
-JS
-  ${pad('initial load', 50)}${pad(kb(js.entryHTML || 0), 8, 'left')}
-  ${pad('total size', 50)}${pad(kb(js.total), 8, 'left')}
-CSS
-  ${pad('initial load', 50)}${pad('0 kB', 8, 'left')}
-  ${pad('total size', 50)}${pad('0 kB', 8, 'left')}
```

Comment:
haha, i can't believe I didn't notice this while working on the build

Reply:
😛 haha shows how many people pay attention to build output! /s